examples/client.yml

```diff
@@ -8,11 +8,14 @@ tls:
   cert: /path/to/client-certificate.pem 
   key: /path/to/client-certificate-key.pem
 
-# OpenSSH host key (unsigned)
-host_key: /etc/ssh/ssh_host_rsa_key.pub
-
-# Where to install the signed host certificate
-signed_cert: /etc/ssh/ssh_host_rsa_key_signed.pub
+# List of host keys for OpenSSH server
+host_keys:
+  # Here, 'key' is the public key, and 'cert' is where to install the signed cert
+  - plain: "/etc/ssh/ssh_host_rsa_key.pub"
+    signed: "/etc/ssh/ssh_host_rsa_key.cert"
```

Comment:
convention in openssh seems to be to use `ssh_host_rsa_key-cert.pub`, I think it would be good to follow that in our examples